### ISSUE TITLE
Auto login on program start

### ISSUE THREAD

**Issue by DragonHeart000** [DragonHeart000/TwitchChatHotKeys#26]
**Description**
To make logging in even more convenient than going to `file > load` the program could have an option to log in for you using saved settings.

**Version**
0.4.0Alpa

**OS**
All

**Screenshot/video**
N/A

**Other Notes**

**Comment by LGajdos10** [comment]
Hi @DragonHeart000 Can I work on  this?

**Comment by DragonHeart000** [comment]
@LGajdos10 Yeah go ahead! Be sure to check out the [Contributing.md](https://github.com/DragonHeart000/TwitchChatHotKeys/blob/master/CONTRIBUTING.md) if you have not yet already. I have not had a bunch of time to work on this project since I do it purely in my free time as a hobby and I recently picked up a few other things I am working on so I would really appreciate the help! I do want to keep this project alive and knowing that others wanna work on it and use it really helps with motivation! Lemme know if you have any questions! The `FileHandleing.java` class should help with reading in and out data and the option to turn on auto-login would be nice under the "More" button on the login screen.
